modules/ROOT/pages/FAQ/index.adoc

```diff
@@ -741,7 +741,10 @@ For more information and examples of metadata driven applications: https://docs.
 
 Is there a roadmap (release plan) for future eForms SDK releases or a set release date for SDK versions?::
 
-The eForms SDK is a complicated development and information is made available as fast as possible. 
+The eForms SDK is a complicated development and information is made available as fast as possible.
+The SDK roadmap is available at link:https://docs.ted.europa.eu/eforms-common/roadmap/index.html[eForms SDK roadmap] 
```

Comment:
We should not put the full URL for links to other pages in the docs.
See https://docs.antora.org/antora/latest/page/xref/